otter/test/models/test_cass_models.py

```diff
@@ -2527,6 +2557,68 @@ def test_add_scaling_policy_cron(self):
         self.assertEqual(result, [pol])
 
 
+class ScalingGroupWebhookMigrateTests(SynchronousTestCase):
+    """
+    Tests for webhook migration functions in :obj:`CassScalingGroup`
+    """
+
+    def setUp(self):
+        """
+        Sample CassScalingGroup object
+        """
+        self.store = CassScalingGroupCollection(None, None)
+
+    def test_webhook_index_only(self):
+        """
+        Test `get_webhook_index_only`
```

Comment:
Well.. I would just have to repeat the docstring of `get_webhook_index_only` here to make sense. Do you think its ok to put `Refer get_webhook_index_only docs` here?